examples/pos/parametrized.effekt

```diff
@@ -0,0 +1,39 @@
+effect State[T] {
+  def get(): T
+  def put[S](n: T, x: S): S
+}
```

Comment:
Here you can see the new feature. It is now possible that effects are parametrized by type parameters.